contracts/dispatcher/src/contract.rs

```diff
@@ -23,7 +23,8 @@ use crate::{
 };
 
 // version info for migration info
-const CONTRACT_VERSION: Version = 0;
+// const EXPECTED_MIGRATION_STORAGE_VERSION: Version = 0;
```

Comment:
[everywhere] please rename to `CONTRACT_STORAGE_VERSION_FROM`

Reply:
Addressed.